invenio/utils/persistentid.py

```diff
@@ -316,6 +334,7 @@ def is_pmcid(val):
     ('ean8', is_ean8),
     ('istc', is_istc),
     ('pmid', is_pmid),
+    ('gnd', is_gnd),
```

Comment:
Isn't that already done in https://github.com/inveniosoftware/invenio/pull/3254/files#diff-f3e79904dc66a9f6d0ebca1960b25154L336?